content/en/docs/concepts/architecture/nodes.md

```diff
@@ -513,6 +513,30 @@ During a non-graceful shutdown, Pods are terminated in the two phases:
   recovered since the user was the one who originally added the taint.
 {{< /note >}}
 
+### Forced storage detach on timeout {#storage-force-detach-on-timeout}
```

Comment:
This is a lot of detail for someone who may be learning, for the very first time, what a Kubernetes cluster is.

We'd do well to find a better home for this detail.

Reply:
How about https://kubernetes.io/docs/concepts/cluster-administration/?

Reply:
I don't know...

I feel like this belongs in the same section as [Non-Graceful node shutdown](https://kubernetes.io/docs/concepts/architecture/nodes/#non-graceful-node-shutdown).

After all, we're saying "This option is risky and can lead to data corruption; how about using the option above?"

Reply:
> I feel like this belongs in the same section as [Non-Graceful node shutdown](https://kubernetes.io/docs/concepts/architecture/nodes/#non-graceful-node-shutdown).

I agree. We should move _both_ of those details to another page, ideally.

Reply:
I believe that that page was probably put there because graceful node shutdown was a thing:

https://github.com/kubernetes/website/blob/snapshot-initial-v1.20/content/en/docs/concepts/architecture/nodes.md#graceful-node-shutdown-graceful-node-shutdown

If we think this is the right thing to do, I'd be happy to move both Graceful & Non-Graceful, just Non-Graceful, or just the new section.

What makes the most sense at a high level?

Reply:
If this merges, I've created an issue to move "Graceful Node Shutdown" and "Ungraceful Node Shutdown" sections along with "Forced storage detach on timeout" (PR pending) section
Issue https://github.com/kubernetes/website/issues/45690